playbooks/roles/scalardl/defaults/main.yml

```
@@ -1,15 +1,16 @@
 local_helm_charts_values_directory: "{{ lookup('env','SCALAR_K8S_CONFIG_DIR') | default('../conf', True) }}"
 remote_directory: "/home/{{ ansible_user }}"
 helm_charts_custom_config_path: "{{ remote_directory }}/helm/config"
-helm_charts_clone_path: "{{ remote_directory }}/helm/helm-charts"
-helm_charts_repo_url: https://github.com/scalar-labs/helm-charts.git
-helm_charts_version: main
+scalardl_charts_repo_url: https://scalar-labs.github.io/helm-charts
+scalardl_charts_repo_name: scalar-labs
+scalardl_charts_version: 1.2.2
+schema_loading_charts_version: 1.3.0
```

[nits] The singular may be appropriate.

```suggestion
schema_loading_chart_version: 1.3.0
```

---

fixed in 84e82d8.